lib/migration_generator/operation.ex

```diff
@@ -1219,29 +1219,29 @@ defmodule AshPostgres.MigrationGenerator.Operation do
 
   defmodule RemoveCustomIndex do
     @moduledoc false
-    defstruct [:schema, :table, :index, :base_filter, :multitenancy, no_phase: true]
+    defstruct [:schema, :table, :index, :base_filter, :multitenancy, :old_multitenancy, no_phase: true]
```

Comment:
Do we need to update where we create this and add `old_multitenancy`?

Reply:
Works like that because of [this line](https://github.com/ash-project/ash_postgres/blob/0124cb4c82f9327c3565103f0a3f135f1292a04f/lib/migration_generator/migration_generator.ex#L2113) that adds `old_multitenancy` field to all operations.